libr/bin/p/bin_wasm.c

```diff
@@ -216,6 +214,8 @@ static RList *imports(RBinFile *bf) {
 		if (!(ptr = R_NEW0 (RBinImport))) {
 			goto bad_alloc;
 		}
+		eprintf("%s\n", import->field_str);
```

Comment:
debug printfs

Reply:
you can now use `R_LOG_DEBUG` if you want :)

Reply:
removed